### PR TITLE
firs api consume test

### PR DESCRIPTION
**Pull request by Daniel2894**

### CODE REVIEW

File: test/MyFirstApiConsume.test.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    it('Consume PUT Service', () => {
+        const body = {
+            name: 'John',
+            age: 31,
+            city: 'New York'
+        };
+
+        return agent.put('https://httpbin.org/put')
+        .send(body).then((response) => {
```
Comment (sorregom) [inline test/MyFirstApiConsume.test.js]: add a tab to this method to keep order

File: test/MyFirstApiConsume.test.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    });
+        
+        it('Consume DELETE Service', () => {
+            const body = {
+                name: 'John',
+                age: 31,
+                city: 'New York'
+            };
+            return agent.del('https://httpbin.org/delete')
+            .send(body).then((response) => {
```
Comment (sorregom) [inline test/MyFirstApiConsume.test.js]: please add a tab to this method to keep order, also the delete method response is OK?

File: test/MyFirstApiConsume.test.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                expect(response.status).to.equal(statusCode.OK);
+                expect(response.body.json).to.eql(body);
+            });
+        });
+
```
Comment (sorregom) [inline test/MyFirstApiConsume.test.js]: please delete this empty line

File: test/MyFirstApiConsume.test.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    it('Consume PATCH Service', () => {
+        const body = {
+            city: 'Miami'
+        };
+        return agent.patch('https://httpbin.org/patch')
+            .send(body).then((response) => {
+                expect(response.status).to.equal(statusCode.OK);
```
Comment (sorregom) [inline test/MyFirstApiConsume.test.js]: patch response is with OK?

File: test/MyFirstApiConsume.test.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+          .then((response) => {
+            expect(response.status).to.equal(statusCode.OK);
+            expect(response.body.json).to.eql(body);
+          });
+      });
+
+      it('Consume HEAD Service', () => {                
+            return agent.head('https://httpbin.org/headers')
+                .then((response) => {
+                    expect(response.status).to.equal(statusCode.OK);
```
Comment (sorregom) [inline test/MyFirstApiConsume.test.js:49]: head response is OK?

File: test/MyFirstApiConsume.test.js [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+            name: 'John',
+            age: 31,
+            city: 'New York'
+        };
+     
+        return agent
+          .post('https://httpbin.org/post')
+          .send(body)
+          .then((response) => {
+            expect(response.status).to.equal(statusCode.OK);
```
Comment (sorregom) [inline test/MyFirstApiConsume.test.js:41]: post response is OK?

**Review by sorregom — CHANGES_REQUESTED** [review]
please check comments

File: test/MyFirstApiConsume.test.js (hunk 5 above)
Comment (Daniel2894) [reply test/MyFirstApiConsume.test.js:49]: Yes, postman return status: 200 OK

File: test/MyFirstApiConsume.test.js (hunk 6 above)
Comment (Daniel2894) [reply test/MyFirstApiConsume.test.js:41]: Yes, postman return status: 200 OK

**Review by sorregom — APPROVED** [review]
LGTM

**Pull request merged**